commands/motor/MotorSet.java

```diff
@@ -50,18 +54,22 @@ protected void execute() {
 	}
 
 	@Override
-	protected void end() {
+	public void end(boolean interrupted) {
 		motor.set(0);
-		LogKitten.d("MotorSet ended (motor speed set to 0)");
+		if (interrupted) {
+			LogKitten.d("MotorSet interrupted?!");
+		} else {
+			LogKitten.d("MotorSet ended (motor speed set to 0)");
+		}
 	}
 
 	@Override
-	protected void interrupted() {
-		LogKitten.d("MotorSet interupted (motor speed undefined)");
+	public boolean isFinished() {
+		return false;
 	}
 
 	@Override
-	protected boolean isFinished() {
-		return false;
+	public Set<Subsystem> getRequirements() {
```

Comment:
delete this method. its already made above

Reply:
also returning `null` randomly like this is probably not the way to go

Reply:
out of scope